delta/imagery/sources/worldview.py

```diff
@@ -101,10 +102,21 @@ def _prep(self, paths):
            TODO: Apply TOA conversion!
         """
         assert isinstance(paths, str)
-        parts = os.path.basename(paths).split('_')
+        (_, ext) = os.path.splitext(paths)
+        if ext == '.zip':
```

Comment:
Will this work with non-zip files? when we call unpack later I thought we assumed it was a zip. (not sure this check is needed)